### PR TITLE
Adding myself in the list of participants!

### PR DESCRIPTION
**Pull request by VaasuCodez**

### CODE REVIEW

**Comment by VaasuCodez** [comment]
Hey its my first PR. I got a message about 4 checks failed. Kindly help

File: src/contributors/VaasuCodez.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Vaasu", 
+    "desc": "Student",
+    "github": "https://github.com/VaasuCodez",
```
Comment (iamdarshshah) [inline src/contributors/VaasuCodez.json:4]:
```suggestion
    "github": "https://github.com/VaasuCodez"
```
Remove this trailing comma or else add your twitter or linkedin profile.

Comment (VaasuCodez) [reply src/contributors/VaasuCodez.json:4]: Thanks a bunch!

**Comment by VaasuCodez** [comment]
Ok I fixed it. Thanks again for the help @iamdarshshah !

**Review by iamdarshshah — APPROVED** [review]
Cool! thanks @VaasuCodez

**Pull request merged**